.github/workflows/pull-request.yaml

```
@@ -0,0 +1,48 @@
+          node-version: latest
+
+      - name: Detect workspace changes
+        id: detect
+        run: |
+          # TODO: Implement change detection logic
+          # Should output:
+          # - changed-workspaces: JSON array of workspaces names
+          # - has-changes: 'true' or 'false'
+          echo "TODO: Run node scripts/detect-changes.mjs"
+
+  set-labels:
```

actionlint (1.7.7)

"steps" section is missing in job "set-labels"

(syntax-check)